client/blocks/reader-post-card/style.scss

```diff
@@ -174,6 +174,7 @@
 
 		.reader-post-card__title-link {
 			display: -webkit-box;
+			line-break: anywhere;
```

Comment:
I've seen this used in the `.reader-post-card__title` it seems to break the word only when neccessary
```suggestion
				overflow-wrap: break-word;
```

Reply:
Oh, sweet. I'll try that. Thanks!

Reply:
I tried `overflow-wrap: break-word;` but it didn't seem to work in all places.